modules/org.restlet.ext.apispark/src/org/restlet/ext/apispark/internal/model/Parameter.java

```diff
@@ -35,24 +32,9 @@
  */
 public class Parameter {
 
-    /**
```

Comment:
This does not change anything but in the class Header in which we add a few properties. 
Not breaking.